src/azul/project/hca/transformers.py

```diff
@@ -208,11 +243,15 @@ def create_documents(self,
                             version=version,
                             manifest=manifest,
                             metadata_files=metadata_files)
+
```

Comment:
In my opinion, even though the method isn't that long, a single newline to separate between assignments, function calls, control structures, return statement etc. would make it quicker to find the code that we want. Here is an example.

```python
# The code is written to illustrate the readability discussion.
def str_concat(s1, s2):
  final_strings : List[str] = list()
  if not isinstance(s1, str):
    final_strings.append(str(s1))
  else:
    final_strings.append(s1)
  final_strings.append(str(s2) if not isinstance(s2, str) else s2)
  return ''.join(final_strings)
```

Now, with the same code with line separations:

```python
# The code is written to illustrate the readability discussion.
def str_concat(s1, s2):
  final_strings : List[str] = list()

  if not isinstance(s1, str):
    final_strings.append(str(s1))
  else:
    final_strings.append(s1)

  final_strings.append(str(s2) if not isinstance(s2, str) else s2)

  return ''.join(final_strings)
```

Even though the method in this example is not long at all, I believe the extra spaces would help developers navigate the code easier due to increase in readability (See [Active vs. passive](https://www.interaction-design.org/literature/article/the-power-of-white-space)) and make the coding style more consistent regardless to the length of the method or function.

@hannes-ucsc, since we don't have written coding guideline, what is your opinion on this?

Reply:
I prefer the former for the reason I mentioned. I am fully aware that this is a matter of preference but as the project lead I made a choice and everyone should stick with that decision.